Gopkg.toml

```diff
@@ -34,7 +34,8 @@
 
 [[override]]
   name = "github.com/mitchellh/go-ps"
-  branch = "master"
+  source = "github.com/gravitational/go-ps"
```

Comment:
Do you think it's worth trying submitting the PR upstream? We'll soon fork all our dependencies :)

Reply:
https://github.com/mitchellh/go-ps/pull/39